src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -2050,6 +2051,7 @@ shared_ptr<runtime::Executable>
         case OP_TYPEID::Elu:
         case OP_TYPEID::EmbeddingLookup:
         case OP_TYPEID::Erf:
+        case OP_TYPEID::FakeQuantize:
```

Comment:
Please add "case OP_TYPEID::FakeQuantize:" line into switch of runtime::intelgpu::IntelGPUBackend::is_supported_impl function (intelgpu_backend.cpp). By #2912 we implemented functionality to decompose unsupported fused ops.